selfdrive/controls/controlsd.py

```diff
@@ -540,7 +547,9 @@ def state_control(self, CS):
         cloudlog.error(f"actuators.{p} not finite {actuators.to_dict()}")
         setattr(actuators, p, 0.0)
 
-    return actuators, lac_log
+    CC.actuators = actuators
```

Comment:
do this the other way around instead of creating an actuators message manually, i.e. do this instead of making a new msg
```suggestion
    actuators = CC.actuators
```